test_conformance/basic/test_progvar.cpp

```diff
@@ -1570,6 +1629,18 @@ int test_progvar_prog_scope_init(cl_device_id device, cl_context context, cl_com
 // A simple test for support of static variables inside a kernel.
 int test_progvar_func_scope(cl_device_id device, cl_context context, cl_command_queue queue, int num_elements)
 {
+    cl_bool skip{ CL_FALSE };
+    auto error = should_skip(device, skip);
+    if (CL_SUCCESS != error)
+    {
+        return TEST_FAIL;
+    }
+    if (skip)
+    {
+        log_info("Skipping progvar_prog_scope_misc since it is optionally not "
+                 "supported on this device");
```

Comment:
Please add `\n` in the end of `log_info` message

Reply:
Fixed.